src/MLJClusteringInterface.jl

```diff
@@ -31,13 +31,14 @@ const PKG = "MLJClusteringInterface"
 @mlj_model mutable struct KMeans <: MMI.Unsupervised
     k::Int = 3::(_ ≥ 2)
     metric::SemiMetric = SqEuclidean()
+    init = :kmpp
 end
 
 function MMI.fit(model::KMeans, verbosity::Int, X)
     # NOTE: using transpose here to get a LinearAlgebra.Transpose object
     # which Kmeans can handle.
     Xarray = transpose(MMI.matrix(X))
-    result = Cl.kmeans(Xarray, model.k; distance=model.metric)
+    result = Cl.kmeans(Xarray, model.k; distance=model.metric, init = model.init)
```

Comment:
As orthogonal to hierarchical clustering, please move this change to a separate PR an include an update to the (recently expanded) doc-string.